sklearn/dummy.py

```diff
@@ -56,11 +56,10 @@ class DummyClassifier(MultiOutputMixin, ClassifierMixin, BaseEstimator):
              Dummy Classifier now supports prior fitting strategy using
              parameter *prior*.
 
-    random_state : int, RandomState instance or None, optional, default=None
-        If int, random_state is the seed used by the random number generator;
-        If RandomState instance, random_state is the random number generator;
-        If None, the random number generator is the RandomState instance used
-        by `np.random`.
+    random_state : int, RandomState instance, default=None
+        Determines random choice of seed.
```

Comment:
I think the point of this issue is to be more explicit. For example, this is used only in the stratified or uniform case. So an appropriate description might be "Determines the random generation of uniform or stratified predictions."